src/cdk/schematics/ng-update/find-stylesheets.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {join} from '@angular-devkit/core';
+import {Tree} from '@angular-devkit/schematics';
+
+/** Regular expression that matches stylesheet paths */
+const STYLESHEET_REGEX = /.*\.(css|scss)/;
+
+/** Finds stylesheets in the given directory from within the specified tree. */
+export function findStylesheetFiles(tree: Tree, baseDir: string): string[] {
+  const result: string[] = [];
+  const visitDir = dirPath => {
+    // Do not visit directories or files inside node modules or `dist/` folders.
+    if (/^\/?(node_modules|dist)\//.test(dirPath)) {
```

Comment:
1. I'm not sure whether this is dealing with absolute or relative paths, but it won't match something like `/foo/bar/node_modules/` or `./node_modules`.
2. This won't match if Windows returns a path like `node_modules\\foo\\bar`. I don't know if that's normalized already.

Reply:
This is actually dealing with workspace paths where separators are always POSIX forward. It's also matching the previous logic where we only skip `node_modules` at the top-level of a project. Maybe we should skip nested node modules too if they exist for whatever reason.